.eslintrc.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
Will you move this back to .eslintrc.js.  Just so we can see the changed rules?

Thank you